userapi/storage/accounts/sqlite3/storage.go

```diff
@@ -22,14 +22,16 @@ import (
 	"strconv"
 	"sync"
 
+	// Import the sqlite3 package
+	_ "github.com/mattn/go-sqlite3"
```

Comment:
We don't technically need this here as we are importing it in other places. It's a little misleading to put this here because we don't always use `mattn/go-sqlite3` - in WASM mode we use https://github.com/matrix-org/go-sqlite3-js

Reply:
then the best way is to just rm the comment as a whole